pkg/filters/filters_test.go

```diff
@@ -185,3 +186,16 @@ func TestComputeUntilTimestamp(t *testing.T) {
 		})
 	}
 }
+
+func TestGetLibpodFilters(t *testing.T) {
+	testFilters := map[string][]string{
+		"label":     {"foo=bar", "xyz"},
+		"reference": {"image-name"},
+	}
+	expectedLibpodFilters := []string{"label=foo=bar", "label=xyz", "reference=image-name"}
+	t.Run("Testing getLibpodFilters", func(t *testing.T) {
+		if got := getLibpodFilters(testFilters); !reflect.DeepEqual(got, expectedLibpodFilters) {
+			t.Errorf("getLibpodFilters() = %v, want %v", got, expectedLibpodFilters)
+		}
```

Comment:
(Absolutely non-blocking: This can probably be shorter with `testify/assert.Equal`. This works fine as is.)

Reply:
Fixed